src/Scanner/TimeNodeScanner.ts

```diff
@@ -68,6 +68,11 @@ export default class TimeNodeScanner extends ChainScanner implements ITimeNodeSc
   }
 
   private async runAndSetInterval(fn: () => Promise<void>, interval: number): Promise<IntervalId> {
+    /* tslint:disable */
+    // if (!this.scanning) {
+    //   this.config.logger.debug('Not starting intervals when TimeNode is intentionally stopped.');
+    //   return null;
+    // }
```

Comment:
Is this supposed to be commented out? Should be removed if no longer used.

Reply:
@adibas03 When testing I found that this code block won't start the scanning the first time since `this.scanning` is flipped to `true` after calling this function. Is there a way to change it and still get the desired effects?